### DISCUSSION TITLE
Broadcast Event & wire:navigate

### DISCUSSION THREAD

**Discussion by codevApp** [livewire/livewire#7567] [1. Help]
I have an app consisting of several components.
A method is executed in the start component using a listener. Works like in Livewire 2.

```
protected function getListeners()
    {
        return [
            'echo:user.'. Auth::id() .'.account,AccountUpdated' => 'accountUpdatedListener',
        ];
    }
```
Everything fine.

But:
I navigate within the app using wire:navigate.
If i switch from the start page to another component via wire:navigate, the next component also reacts on the listener from the first loaded compontent (start page) with an error in the console:

**Uncaught Component not found: mNPFHWMGEwJmTbDpzbxg**

It works without wire:navigate. 
It seems as if the listener is hold but when the page is changed the component is logically no longer there.

Does anyone have any advice?

**Comment by joshhanley** [comment]
@codevApp it's likely that there is a bug when it comes to `wire:navigate` and echo. Seems like it's not cleaning up the event listeners properly.

**Reply by codevApp** [reply]
@joshhanley yes, found a bug report from @loxi5 at https://github.com/livewire/livewire/discussions/7397
So we can close it here. Will it be fixed? I need it actually.

**Reply by outlawz-dev** [reply]
Hello, any updates on the bug report from @loxi5? It seems to have been removed.
